CONTRIBUTING.md

```diff
@@ -96,56 +96,190 @@ CUDA/GPU requirements:
 You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
 
 
-# Script to build cuCIM from source
+# Building and Testing cuCIM from Source
 
-### Build from Source
+First, please clone the cuCIM's repository
 
-- Clone the repository
 ```bash
 CUCIM_HOME=$(pwd)/cucim
 git clone https://github.com/rapidsai/cucim.git $CUCIM_HOME
 cd $CUCIM_HOME
 ```
+## Local Development using Conda Environment (for gcc 9.x and nvcc 11.0.x)
+
+Conda can be used to setup GCC 9.x, CUDA Toolkit (including nvcc) 11.0.x, and other dependent libraries (as shown in `./conda/environments/env.yml`) for building cuCIM.
+
+Otherwise, you may need to install (such as zlib, xz, yasm) through your OS's package manager (`apt`, `yum`, and so on).
```

Comment:
```suggestion
Otherwise, you may need to install dependencies (such as zlib, xz, yasm) through your OS's package manager (`apt`, `yum`, and so on).
```